### PR TITLE
Add incredibly vital endpoint to the API (/ping)

### PR DESCRIPTION
**Pull request by trentv4**
http://i.imgur.com/PqLu5aA.png

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -65,6 +65,10 @@ Database.then((database) => {
         res.json({ status: 404, error: 'Not Found', message: 'No such endpoint.' });
     });
 
+    app.get('/ping', (req, res) => {
+        res.send('pong');
+    });
+
```
Comment (JamiesWhiteShirt) [inline server.js]: This very much needs unit testing.

File: server.js [hunk 2]
```diff
@@ -65,6 +65,10 @@ Database.then((database) => {
         res.json({ status: 404, error: 'Not Found', message: 'No such endpoint.' });
     });
 
+    app.get('/ping', (req, res) => {
```
Comment (JamiesWhiteShirt) [inline server.js]: Preferred method would be POST, not GET.

**Comment by JamiesWhiteShirt** [comment]
We currently lack the technology to make it work properly like a ping-pong. Due to the importance of this feature it is best that we add this anyway.

File: server.js [hunk 3]
```diff
@@ -65,6 +65,10 @@ Database.then((database) => {
         res.json({ status: 404, error: 'Not Found', message: 'No such endpoint.' });
     });
 
+    app.post('/ping', (req, res) => {
```
Comment (TheCodedOne) [inline server.js]: Change to all and i'll be happy

**Pull request merged**